man/org.freedesktop.systemd1.xml

```diff
@@ -1472,12 +1472,12 @@ node /org/freedesktop/systemd1 {
       <para>Read access is generally granted to all clients. Additionally, for unprivileged clients, some
       operations are allowed through the polkit privilege system. Operations which modify unit state
       (<function>StartUnit()</function>, <function>StopUnit()</function>, <function>KillUnit()</function>,
-      <function>RestartUnit()</function> and similar, <function>SetProperty</function>) require
+      <function>RestartUnit()</function>) and similar, <function>SetProperty()</function> require
```

Comment:
All those functions modify unit state, i.e. they should all be parenthesized together.